diff --git a/node_modules/@adrianso/react-native-device-brightness/android/build.gradle b/node_modules/@adrianso/react-native-device-brightness/android/build.gradle
index a9d7d54..39bd498 100644
--- a/node_modules/@adrianso/react-native-device-brightness/android/build.gradle
+++ b/node_modules/@adrianso/react-native-device-brightness/android/build.gradle
@@ -5,7 +5,7 @@ android {
     buildToolsVersion "25.0.2"
 
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion 21
         targetSdkVersion 22
         versionCode 2
         versionName "1.1"
